.travis.yml

```diff
@@ -16,6 +16,8 @@ before_install:
   - go get -u github.com/pressly/goose/cmd/goose
   - $HOME/gopath/bin/goose -dir migrations mysql "root@/comedian" up
   - go get github.com/mattn/goveralls
+  - go get github.com/go-chi/chi
```

Comment:
We have to install goveralls because it's binary and not located in GOPATH. chi and logrus should be vendored. We don't have to install them before running travis build. Please use dep for vendoring